src/libsystemd/sd-id128/id128-util.c

```diff
@@ -13,6 +13,23 @@
 #include "string-util.h"
 #include "sync-util.h"
 
+int id128_from_string_not_null(const char *s, sd_id128_t *ret) {
```

Comment:
hmm, maybe call this `id128_from_string_refuse_null()? otherwise people might thing this was about null pointers, but it is really about the null uuid?

Reply:
I'm not sure if "refuse_null" is better than "not_null" — it's "null" that is confusing, not the other word. I changed it to `id128_from_string_nonzero`. I hope that's better.